BuildAndTest.proj

```diff
@@ -75,13 +76,17 @@
 
       <!-- MakeConst unit tests tracked by https://github.com/dotnet/roslyn/issues/5918 -->
       <TestAssemblies Condition="'$(Test64)' != 'true'" 
-                      Include="$(OutputDirectory)\**\$(IncludePattern)" />
+                      Include="$(OutputDirectory)\$(IncludePattern)" />
```

Comment:
It's in future, under https://github.com/dotnet/roslyn/tree/future/src/VisualStudio/ProjectSystem. 

You'll need to change to change DeployTestDependencies and the test projects (currently deploy to bin\ProjectSystem\Tests). The product itself deploys into a different folder (bin\ProjectSystem) - so don't change that as you'll cause a build race.

It might be easier to do what I suggested above, and then I'll/you'll fixed it future. I just want to avoid accidentally merging this into future and then tests stop running.